docs/vulsrepo.md

```diff
@@ -240,7 +243,8 @@ When the data is loaded, the `pivot table` is shown as follows.
 If you try to load a large amount of data, it may fail. Please use the following methods.
 
 - Reduce the selection of the data to be displayed and load it again.
-- Setting `Summary` and `Cvss Metrics` to `OFF` in the `Setting` may solve this problem (see [Display setting - Show / Hide the Item](#show--hide-the-item)).
+- Make sure that you have not changed the directory name and file name under the `reselts` directory.
```

Comment:
There is a minor typo here, `reselts` intead of `results` I guess.

Reply:
Thank you. I will fix it.

Reply:
You're welcome, glad I could help :)